doc/rados/configuration/mon-config-ref.rst

```diff
@@ -770,6 +770,41 @@ Client
 :Default: ``100ul << 20``
 
 
+Pool settings
+=============
```

Comment:
@wido some of these settings are also documented at http://docs.ceph.com/docs/master/rados/operations/pools/#set-pool-values, would be ideal if you could add references to them?